README.md

```diff
@@ -35,6 +35,7 @@ modelerfour:
   flatten-payloads: true
   resolve-schema-name-collisons: true
   always-create-content-type-parameter: true
+  multiple-request-parameter-flattening: false
```

Comment:
With this change, can we get rid of our code in tasks.py that prevents payload-flattening-threshold having a value when generating multiple media types: https://github.com/Azure/autorest.python/blob/788744b91f5b4b265b7ed2f99b771aa43f9b9ece/tasks.py#L130

Reply:
If so, I can write the code and push it to remove thi

Reply:
Excellent, I was wondering how to test that, I think you're right, removing this should generate the same code or it means it's not fixed. Good catch!